### ISSUE TITLE
Feature Request: receive email newsletters

### ISSUE THREAD

**Issue by mockdeep** [swanson/stringer#575]
I heard about [this feature in Inoreader](https://www.inoreader.com/blog/2020/02/declutter-your-inbox-subscribe-to-email-newsletters-straight-into-inoreader.html), where you can subscribe your reader to an email newsletter rather than needing to receive it in your personal email. Seems like a pretty handy thing, especially for those newsletters that are themselves a bundle of links.

**Comment by jhass** [comment]
I do it through https://notifier.in

**Comment by gabrielpoca** [comment]
I've been using this one https://kill-the-newsletter.com/ turn an email inbox into a RSS feed

**Comment by giovapanasiti** [comment]
This could be easily done integration action mailbox but then you should be doing some manual setup anyway. What you think @mockdeep ?

**Comment by mockdeep** [comment]
@giovapanasiti yeah, I think ActionMailbox is probably a good place to start. I'm not sure about the details, though. For example, Stringer currently relies heavily on a feed that it can query for, but in this case it would presumably be a no-op for fetching the feed. Also, currently stories typically have links you can visit to see the story on the external site, but that wouldn't be the case with these.

In principle, I'd love aggregator newsletters to be split into separate stories, like when I receive Ruby Weekly I would want it to create a separate story for each item in the email. Not sure how much work that would be, though. Probably something that wouldn't need to go in the first version of the feature.

At any rate, if you want to give it a try, I'd be happy for the contribution!

**Comment by giovapanasiti** [comment]
I'll try to figure something out and will post here some more documentation before coding for the solution.
